test/validation/api/init/Makefile.am

```diff
@@ -3,7 +3,8 @@ include ../Makefile.inc
 # Keep init test cases in separate binaries. Some implementations may not allow
 # the same application process to call odp_init_global() multiple times.
 test_PROGRAMS = init_defaults init_abort init_log init_num_thr \
-		init_feature_enabled init_feature_disabled init_log_thread
+		init_feature_enabled init_feature_disabled init_log_thread \
+		init_test_param_init
```

Comment:
The new binary should be added to the folder's `.gitignore` and added to `TESTS` of `/test/validation/api/Makefile.am`.

Reply:
Thanks @TuomasTaipale . Added changes as per suggestion.